addons/website_sms/views/website_visitor_views.xml

```diff
@@ -21,7 +21,7 @@
                 <field name="mobile" widget="phone"/>
             </field>
             <xpath expr="//div[hasclass('w_visitor_kanban_actions')]" position="inside">
-                <button name="action_send_sms" type="object" class="btn btn-secondary"
+                <button name="action_send_sms" type="object" class="btn btn-secondary me-1"
```

Comment:
I would not play with margins here. As a matter of fast, I don't think you added it on the correct element. As far as I can see, the SMS button is the last one, so it should either have ms-1, or the Chat button should have me-1.

But as I said, I wouldn't use margins. Instead, I would
 - replace class `w_visitor_kanban_actions` by a name (there's no associated style) [this is bonus, not really part of the fix]
 - add class `d-flex gap-1` on that node
 - remove class me-1 from the Email button

Don't forget to re-target to v18